### DISCUSSION TITLE
As of about a month ago the bot has not worked at all.

### DISCUSSION THREAD

**Discussion by Viinyl** [jagrosh/MusicBot#1596] [Questions]
I'm using v0.4.0, it launches and everything seems to work fine, it'll join the VC we're in but refuses to play anything, it's never had any issues beforehand. upon downloading 0.4.1 the actual .jar itself never loads and the bot in turn does not run at all. Just wondering what may have changed?

none of my settings have been reset or changed and my hardware remains the same as it's been since I first downloaded the bot. Unsure as to what's happened just wondering if anyone else has been having this issue.

**Comment by MichailiK** [comment]
Starting 0.4.1, JMusicBot now requires **Java 11** to be used. Install Java 11 & use it to run JMusicBot.

You may still encounter a separate issue, #1588 with YouTube tracks, as YouTube/Google have been seriously cracking down on 3rd party apps like JMusicBot.

**Reply by Br4inD3ad** [reply]
I'm new at this but it was working on 0.3.9. I updated all of the files required and yet I keep getting this message. I'm not sure how to update to Java 11 through Pebblehost. Should I contact them and have them?? Thank you!

[Bot] Exception in thread "main" java.lang.UnsupportedClassVersionError: com/jagrosh/jmusicbot/JMusicBot has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
